ui/src/shared/utils/sort.ts

```diff
@@ -0,0 +1,42 @@
+import {orderBy, get, toLower} from 'lodash'
+
+export enum SortTypes {
+  String = 'string',
+  Date = 'date',
+  Float = 'float',
+}
+
+export const sortSelector = (_, props) => ({
```

Comment:
are you still using this anywhere? couldn't see it.

Reply:
No, I deleted it now